media-video/mjpg-streamer/mjpg-streamer-0_pre20120621-r2.ebuild

```diff
@@ -13,37 +13,40 @@ LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 
-INPUT_PLUGINS="input_testpicture input_control input_file input_uvc"
-OUTPUT_PLUGINS="output_file output_udp output_http output_autofocus output_rtsp"
+INPUT_PLUGINS="inputtestpicture inputcontrol inputfile inputuvc"
+OUTPUT_PLUGINS="outputfile outputudp outputhttp outputautofocus outputrtsp"
 IUSE_PLUGINS="${INPUT_PLUGINS} ${OUTPUT_PLUGINS}"
-IUSE="input_testpicture input_control +input_file input_uvc output_file
-	output_udp +output_http output_autofocus output_rtsp
+IUSE="inputtestpicture inputcontrol +inputfile inputuvc outputfile
+	outputudp +outputhttp outputautofocus outputrtsp
 	www v4l"
 REQUIRED_USE="|| ( ${INPUT_PLUGINS} )
 	|| ( ${OUTPUT_PLUGINS} )
-	v4l? ( input_uvc )"
+	v4l? ( inputuvc )"
 
 RDEPEND="virtual/jpeg
-	v4l? ( input_uvc? ( media-libs/libv4l ) )"
+	v4l? ( inputuvc? ( media-libs/libv4l ) )"
```

Comment:
what's the point of introducing `v4l` flag, while we can just have `inputuvc`?

Reply:
AFAIR inputuvc does not hard require v4l. Simplification would be to meld this requirement together. Might consider doing that.